stable/storageos-operator/questions.yml

```diff
@@ -86,29 +88,20 @@ questions:
     label: Username
   - variable: cluster.admin.password
     default: ""
-    description: "Password of the StorageOS administrator account.  If empty, a random password will be generated."
+    description: "Password of the StorageOS administrator account. Must be at
+    least 8 characters long"
     type: password
     label: Password
 
-  # Telemetry.
-  - variable: cluster.disableTelemetry
-    default: false
-    type: boolean
-    description: "Disable telemetry data collection.  See https://docs.storageos.com/docs/reference/telemetry for more information."
-    label: Disable Telemetry
-
-  # KV store backend.
-  - variable: cluster.kvBackend.embedded
-    default: true
-    type: boolean
-    description: "Use embedded KV store for testing.  Select false to use external etcd for production deployments."
-    label: "Use embedded KV store"
   - variable: cluster.kvBackend.address
-    default: "10.0.0.1:2379"
-    description: "List of etcd targets, in the form ip[:port], separated by commas.  Prefer multiple direct endpoints over a single load-balanced endpoint.  Only used if not using embedded KV store."
+    required: true
+    default: "etcd-client.etcd.svc.cluster.local:2379"
```

Comment:
I think it would add clarity to add the protocol, so when users add TLS for ETCD, they don't forget to make it https. 
so

`http://etcd-client.etcd...`